src/frontend/src/lib/utils/network.utils.ts

```diff
@@ -37,3 +39,10 @@ export const filterTokensForSelectedNetwork = <T extends Token>([
 			$selectedNetwork?.id === networkId
 		);
 	});
+
+export const mapToSignerBitcoinNetwork = ({
+	network
+}: {
+	network: BitcoinNetwork | 'regtest';
```

Comment:
most probably, you will use this notation more: better to already create a type with that, no?

Reply:
You're totally right, it's done now.